tokens/movistar.json

```diff
@@ -1569,6 +1605,14 @@
         "value": "#E63780",
         "type": "color"
       },
+      "pink45": {
+        "value": "#EB5F99",
+        "type": "color"
+      },
+      "pink55": {
+        "value": "#EB5F99",
```

Comment:
this is wrong! the color is repeated. Which is the correct one?

Reply:
`pink55` seems to be `#C42F6D`

Reply:
Alex is right